RecoLocalCalo/HGCalRecProducers/python/HGCalRecHit_cfi.py

```diff
@@ -75,6 +75,13 @@
     HGCHEF_fCPerMIP = HGCalUncalibRecHit.HGCHEFConfig.fCPerMIP,
     HGCHEB_keV2DIGI = hgchebackDigitizer.digiCfg.keV2MIP,
     HGCHEB_isSiFE   = HGCalUncalibRecHit.HGCHEBConfig.isSiFE,
+    # don't produce rechit if detid is a ghost one
+    # smallest and largest ghost detid on positive side
+    GhostDetIdPosMin = 1162362881,
+    GhostDetIdPosMax = 1162362952,
```

Comment:
capitalization rule applies to python configs as well, although it is not enforced particularly strictly